MC/config/PWGHF/ini/GeneratorHFTrigger_ccbar.ini

```diff
@@ -1,20 +1,7 @@
 ### The external generator derives from GeneratorPythia8.
-### This part configures the bits of the interface: configuration and user hooks
+[GeneratorExternal]
+fileName=${O2DPG_ROOT}/MC/config/PWGHF/pythia8/generator/generator_pythia8_gaptriggered_hf.C
```

Comment:
That does not exist

Reply:
That would have been in the `o2sim_serverlog`
I will update the test script to print that. So the error was acknowledged but not printed. Sorry

Reply:
No problem and good catch! Fixed now :)